### PR TITLE
ER-808: Added app warning module

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ER-808

Adds a simple module for showing a warning when using the old app.

### CODE REVIEW

File: breol_app_warning/js/dialog.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+; jQuery(function () {
+  if (Drupal.settings.breol_app_warning) {
```
Comment (cableman) [inline breol_app_warning/js/dialog.js:2]: Should this have been `if (Drupal.settings. hasOwnProperty('breol_app_warning'))` just to be sure to be sure.

File: breol_app_warning/breol_app_warning.info [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name = BReol app warning
+description = "Show warning about obsolete app"
+core = 7.x
+package = eReolen for kids
```
Comment (cableman) [inline breol_app_warning/breol_app_warning.info:4]: Should have dependency on `ding_popup`

File: breol_app_warning/js/dialog.js (hunk 1 above)
Comment (rimi-itk) [reply breol_app_warning/js/dialog.js:2]: This is the way it's usually done throughout Drupal (and JavaScript in general). We could use `hasOwnProperty`, but then we'd need an additional check to see if it's also truthy.

File: breol_app_warning/breol_app_warning.info (hunk 2 above)
Comment (rimi-itk) [reply breol_app_warning/breol_app_warning.info:4]: Nice catch. Fixed!

File: breol_app_warning/js/dialog.js (hunk 1 above)
Comment (cableman) [reply breol_app_warning/js/dialog.js:2]: But would JS not stop execution if the property is not there?

Comment (cableman) [reply breol_app_warning/js/dialog.js:2]: Okay, testet it and it works... but it's not pretty.

**Pull request merged**